openmetadata-ui/src/main/resources/ui/public/locales/en-US/Pipeline/Dagster.md

```diff
@@ -3,7 +3,11 @@
 In this section, we provide guides and references to use the Dagster connector.
 
 ## Requirements
-<!-- to be updated -->
+
+OpenMetadata is integrated with dagster upto version [1.0.13](https://docs.dagster.io/getting-started) and will continue to work for future dagster versions.
+
+The ingestion framework uses [dagster graphql python client](https://docs.dagster.io/_apidocs/libraries/dagster-graphql#dagster_graphql.DagsterGraphQLClient) to connect to the dagster instance and perform the API calls.
+
```

Comment:
Any more details on the endpoint a user need access to? I think we also need to add this to the index.md file.

Reply:
no permissions as such.. just the access token that they bring in..I have added the same under token section